### ISSUE TITLE
loss of the courseplay route and decrease of FPS

### ISSUE THREAD

**Issue by ontot** [Courseplay/courseplay#5236]
Greetings to all. I would like to know if this effect can be fixed. From the next version to the 362 this problem always happens, Every vehicle (tractor or truck) as they walk on the road begin to swing until they get off the road. In addition, from version 397 the game suddenly has a sudden drop in ftp up to 7 FPS for about 10 seconds ... then resumes normal 60 FPS and then still has the drop after about 5 minutes. This problem is always there, throughout the game session. Thanks. Here you can see the problem. https://www.youtube.com/watch?v=D6sE40fNkiQ&feature=youtu.be

**Comment by Tensuko** [comment]
logfile ? #5055

**Comment by ontot** [comment]
Ok ... after some tests and some tests I eliminated the straw dlc which was very annoying to the courseplay. We await the update of the straw DLC. Now the problem of the drop in FPS remains. I only noticed this problem when the courseplay has to "think" where to go. This problem with version 362 is not there, but with version 399 yes. I attach the video.
https://www.youtube.com/watch?v=0y7i6zEszIk&feature=youtu.be 

I also attach the log file 
[log.txt](https://github.com/Courseplay/courseplay/files/4533219/log.txt)
